tracer/src/Datadog.Trace/Configuration/ExporterSettings.cs

```diff
@@ -6,6 +6,7 @@
 using System;
 using System.IO;
 using Datadog.Trace.Agent;
+using Datadog.Trace.Vendors.Serilog;
```

Comment:
Not sure why this has appeared, but it almost certainly shouldn't be here 😉

Reply:
I thought it's what is used to Log early when we're in the setup phase.  cause it's what is used in `TracerSettings` (cf https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/src/Datadog.Trace/Configuration/TracerSettings.cs#L475). I kinda recalled a discussion saying I shouldn't use a Logger early in the setup, but I haven't checked if that's true.

Reply:
Fixed in 96dda0983f4eb26e0bc12bb0078c855df838b7c4. 
Switched to the same system you used in DirectLogsSubmission. ie storing the data in a list appended in the diagnosis log.